lib/Doctrine/DBAL/Migrations/Migration.php

```diff
@@ -118,12 +118,11 @@ public function setNoMigrationException($noMigrationException = false)
      * @param string  $to             The version to migrate to.
      * @param boolean $dryRun         Whether or not to make this a dry run and not execute anything.
      * @param boolean $timeAllQueries Measuring or not the execution time of each SQL query.
-     *
-     * @return array $sql     The array of migration sql statements
-     *
```

Comment:
please keep these lines separating params from the return value. It makes things more readable IMO